server/api/trend.go

```diff
@@ -89,14 +89,16 @@ func newTrendHandler(s *server.Server, rd *render.Render) *trendHandler {
 // @Failure  500  {string}  string  "PD server failed to proceed the request."
 // @Router   /trend [get]
 func (h *trendHandler) GetTrend(w http.ResponseWriter, r *http.Request) {
-	var from time.Time
-	if fromStr := r.URL.Query()["from"]; len(fromStr) > 0 {
-		fromInt, err := strconv.ParseInt(fromStr[0], 10, 64)
+	var (
+		from time.Time
+		err  error
+	)
+	if froms := r.URL.Query()["from"]; len(froms) > 0 {
```

Comment:
Do we need to maintain this interface?

Reply:
I am not sure, this pr only keep the same with pd mode.